server/planning/feeding_services/onclusive_api_service.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from datetime import datetime
+import requests
+from superdesk.io.feeding_services.http_base_service import HTTPFeedingServiceBase
+from flask import current_app as app
+
+logger = logging.getLogger(__name__)
+
+
+class OnclusiveApiService(HTTPFeedingServiceBase):
+    """
+    Feeding Service class which can read events using HTTP
+    """
+
+    NAME = "Onclusive_api"
+    label = "Onclusive api feed"
+    service = "events"
+
+    fields = []
+
+    HTTP_AUTH = False
+    session = requests.Session()
+
+    current_date = datetime.now()
```

Comment:
if you store this on the class it will be the same from the application start till it's restarted, it should be set in in the _update method to be up to date